### PR TITLE
feat: add storybook project with snapshot and e2e testing

### PR DESCRIPTION
**Pull request by nekosaur**
## Description
adds storybook project with snapshot and e2e testing using applitools and testcafe

you need to set `APPLITOOLS_API_KEY` env var to be able to run tests.

there are a couple of different ways to create stories showcased.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/storybook/package.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "storybook",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "start-storybook -p 6006",
+    "watch": "yarn start",
+    "foo": "concurrently --no-color -n snapshot,e2e \"yarn test:snapshot\" \"yarn test:e2e\"",
```
Comment (sh7dm) [inline packages/storybook/package.json]: it probably needs another name

Comment (nekosaur) [reply packages/storybook/package.json]: yeah it was named test before, but then it got triggered on each commit which is unnecessary I think.

Comment (sh7dm) [reply packages/storybook/package.json]: what's about `test:all`?

**Comment by nekosaur** [comment]
@johnleider your changes removed the ability to use .vue files with story blocks?

**Comment by johnleider** [comment]
I was starting to integrate lessons learned from the storybook plugin I’ve been working on but you can revert back to what you had working and get this in.

**Comment by johnleider** [comment]
I was starting to integrate lessons learned from the storybook plugin I’ve been working on but you can revert back to what you had working and get this in.

**Comment by johnleider** [comment]
Not sure if this has to be a feature though as it doesn't touch Vuetify.

**Comment by naveenkash** [comment]
Does this replace `kitchen` and also is the Storybook the way to be taken ahead?

**Comment by MajesticPotatoe** [comment]
> Does this replace `kitchen` and also is the Storybook the way to be taken ahead?

Yes this will be replacing kitchen, and will be the way moving forward.

**Comment by johnleider** [comment]
I'm going to close this for now. I wanted to take this a certain direction and it never came to fruition. I expect this will not be implemented before v3 and I think it's good to push it back there. WHen that time comes, we can create a new PR.

**Pull request closed without merging**